frontend/src/views/web/partials/sidebar.html

```diff
@@ -1,6 +1,6 @@
 <!-- define all main links here for navigation -->
-<div class="ev-sidebar ev-fixed z-depth-3 fixed" id="sidebarToggle">
-    <div class="side-intro ev-md-container fixed">
+<div class="ev-sidebar ev-fixed  fixed hide-on-med-and-down">
+    <!--    <div class="side-intro ev-md-container fixed">
```

Comment:
We might need this later. So let's have this commented code for sometime, will change it later.